### ISSUE TITLE
whitelist bypass through view-source scheme

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#246]
```
imported trac ticket
created: 2011-07-06 16:23:34
reporter: justin
```

[https://grepular.com/ Mike Cardwell] reported another whitelist bypass. This one is through destination URLs with the "view-source" scheme.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-07-06 16:58:22
author: justin
```

Fixed in r444. This will be included in RP 0.5.22.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-07-06 18:36:09
author: justin
```

This fix included in the beta version of 0.5.22, available here:

https://www.requestpolicy.com/releases/requestpolicy-0.5.22b1.xpi
